frontend/javascripts/admin/dataset/dataset_upload_view.js

```diff
@@ -557,6 +557,7 @@ class DatasetUploadView extends React.Component<PropsWithFormAndRouter, State> {
               name="zipFile"
               label="Dataset"
               hasFeedback
```

Comment:
For reference, if you ever want to hide the validation icon you can simply remove this `hasFeedback` prop, no custom CSS needed, I think :)